ml/train.py

```
@@ -16,3 +22,214 @@
+'''
+
+# Remover dados duplicados, se houver.
+df = df.drop_duplicates()
+
+# Tratar outliers, se necessário, substituindo valores muito extremos pela mediana.
+numeric_cols = ['mes_venda', 'qtd_garrafas', 'desconto_aplicado']
+
+for col in numeric_cols:
+    median = df[col].median()
+    df[col] = df[col].apply(
+        lambda x: median
```

Ruff (0.15.13)

[warning] Function definition does not bind loop variable `median`

(B023)